demo/apisix/conf/apisix.yaml

```diff
@@ -162,4 +158,8 @@ plugin_configs:
           failures: 3  # should be var: JMAP_CIRCUIT_BREAKER_MAXERRORS
         healthy:
           successes: 1
+      ext-plugin-pre-req:
+        conf:
+          - name: TokenRevokedFilter
+            value: '{"enable":"feature"}'
```

Comment:
Declaring the token revoked plugin in `jmap-plugin` would enforce the basic routes to revoke the jwt token as well (which is not needed).
Maybe this one should be explicitly declared?

Reply:
I got your meaning, 
Now I'm confusing: 
- Each route has only one `plugin_config_id`, if we declare  `ext-plugin-pre-req` for each route, it is quite verbose
- The `TokenRevokedFilter` will filter requests has authorization header start with `Bearer`, is it fine? IMO it is no problem

Reply:
Rethinking the basic routes is basically for testing purposes. I think it is fine too. As long as this does not break the basic routes flow.

Reply:
Did you try to add an other `plugin_config_id` field in the routes? I didn't test but I would not be surprised it works :)

Reply:
I tried to with below syntax, and it does not work
```yaml
    plugin_config_id: 
      - jmap-plugin
      - other-plugin
```

Reply:
What about this:

```
plugin_config_id: jmap-plugin
plugin_config_id: other-plugin
```
? As I see we can put `plugins` and add a `plugin_config_id` that is defining something similar, maybe can put more than one field :) Maybe not :)

Reply:
NO, it is not a valid yaml syntax